ImprovedAnvil/lang/English/setup.tra

```diff
@@ -7615,4 +7616,466 @@ Armor Class: 4 (6 vs. piercing and missile)
 Requires:
  6 Strength
 
-Weight: 28~
\ No newline at end of file
+Weight: 28~
+
+@12080 = ~Weaken Undead~
```

Comment:
could you start numbering your additions at 13000 and so forth? Or at least 12500. I'm using the same number space in the druid stronghold, there would be pain later on if they overlap.

Reply:
all descriptions below might benefit from some stylistic editing, but we won't bother with it while in beta

Reply:
Okay, I can change them if the installer allows to make such gaps (as there won't be any entries for some 12XXX numbers)

Reply:
yes, the installer won't mind, it just takes @-variables and substitutes their values where necessary, there's no intrinsic requirement to maintain precise ascending numbering

Reply:
Ok, I think it's OK now, updated my strings to start from 13000